boto/ec2/elb/__init__.py

```diff
@@ -756,3 +757,71 @@ def detach_lb_from_subnets(self, name, subnets):
                                'Subnets.member.%d')
         return self.get_list('DetachLoadBalancerFromSubnets',
                              params, None)
+
+    # Tag methods
+
+    @staticmethod
+    def _build_tag_parameters(tags):
+        params = {}
+        for idx, key in enumerate(sorted(tags.keys()), start=1):
+            params['Tags.member.%d.Key' % idx] = key
+            value = tags[key]
+            if value is not None:
+                params['Tags.member.%d.Value' % idx] = value
+        return params
+
+    def get_all_tags(self, load_balancer_names=None):
```

Comment:
Will do.  (I'm currently without my laptop... but will get to this in about 1 week.  Just lucky timing I happened to catch your comment!)